### PR TITLE
doc improvement: aws_fota: Extend troubleshooting section

### PR DESCRIPTION
**Pull request by simensrostad**
Include section regarding increased memory usage
when using presigned URLs.

Removed modem nRF91 modem specific section from
the library doc. The library docs are chip agnostic.

NCSDK-26360

### CODE REVIEW

File: doc/nrf/libraries/networking/aws_fota.rst [hunk 1]
```diff
@@ -113,6 +110,15 @@ Content range is not defined:
...
    Also, confirm that your file is available from the browser without being logged into your AWS account.
 
+Presigned URLs
+--------------
+
+When using presigned URLs it might be the needed to increase the value of:
+ - :kconfig:option:`CONFIG_DOWNLOAD_CLIENT_MAX_FILENAME_SIZE`
+ - :kconfig:option:`CONFIG_AWS_FOTA_PAYLOAD_SIZE`
+ - :kconfig:option:`CONFIG_MBEDTLS_HEAP_SIZE` - If running MBED TLS on the application core. (Wi-Fi builds)
+to accommadate for the increased file name and payload size of the presigned URL, and secure download of the image.
```
Comment (divipillai) [inline doc/nrf/libraries/networking/aws_fota.rst:120]:
```suggestion
When using the presigned URLs, you might need to increase the value of the following Kconfig options to accommodate the long file name and payload size of the presigned URL and the secure download of the image:

 - :kconfig:option:`CONFIG_DOWNLOAD_CLIENT_MAX_FILENAME_SIZE`.
 - :kconfig:option:`CONFIG_AWS_FOTA_PAYLOAD_SIZE`.
 - :kconfig:option:`CONFIG_MBEDTLS_HEAP_SIZE` - If running MBED TLS on the application core (Wi-Fi® builds).
```
This needs to be moved under the [Configuration](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/libraries/networking/aws_fota.html#configuration) section.

**Review by SeppoTakalo — APPROVED** [review]

**Review by bjda — APPROVED** [review]

File: doc/nrf/libraries/networking/aws_fota.rst [hunk 2]
```diff
@@ -113,6 +110,15 @@ Content range is not defined:
    To fix this problem, configure the host server to provide this field.
    Also, confirm that your file is available from the browser without being logged into your AWS account.
 
+Presigned URLs
+--------------
+
+When using presigned URLs it might be the needed to increase the value of:
+ - :kconfig:option:`CONFIG_DOWNLOAD_CLIENT_MAX_FILENAME_SIZE`
+ - :kconfig:option:`CONFIG_AWS_FOTA_PAYLOAD_SIZE`
+ - :kconfig:option:`CONFIG_MBEDTLS_HEAP_SIZE` - If running MBED TLS on the application core. (Wi-Fi builds)
```
Comment (jtguggedal) [inline doc/nrf/libraries/networking/aws_fota.rst:119]:
```suggestion
 - :kconfig:option:`CONFIG_MBEDTLS_HEAP_SIZE` - If running MbedTLS on the application core. (Wi-Fi builds)
```

**Review by jtguggedal — APPROVED** [review]

File: doc/nrf/libraries/networking/aws_fota.rst (hunk 2 above)
Comment (peknis) [reply doc/nrf/libraries/networking/aws_fota.rst:119]: Actually, Mbed TLS.

**Review by peknis — APPROVED** [review]

**Review by jfischer-no — CHANGES_REQUESTED** [review]
Looks like second commit should be squashed.

**Review by divipillai — APPROVED** [review]

**Review by jfischer-no — APPROVED** [review]

**Pull request merged**